### PR TITLE
Moved autogenerated types to be under autogen directory

### PR DESCRIPTION
**Pull request by alonstern**
Moved everything in Netsuite adapter that is autogenerated by `types_generator.py` to be under autogen directory.

Separated the types that are extracted from the NetSuite documentation from the hardcoded functions and types the script would generate to make it easier to edit existing functions or add new ones to `types.ts`

---
_Release Notes_: 
None

---
_User Notifications_: 
None

### CODE REVIEW

File: packages/netsuite-adapter/src/autogen/types/enums.ts [hunk 1]
```diff
@@ -301,7 +301,7 @@ export const enums: Record<string, PrimitiveType> = {
...
...ENUERECOGNITION', 'REVRECSALESORDERFORECASTING', 'REVRECVSOE', 'RFQ', 'RULEBASEDRECOGNITIONTREATMENT', 'SALESCAMPAIGNS', 'SALESCHANNELALLOCATION', 'SALESORDERS', 'SAMLSSO', 'SERIALIZEDINVENTORY', 'SERVERSIDESCRIPTING', 'SERVICEPRINTEDCHECKS', 'SERVICEPRINTEDW2S', 'SFA', 'SHIPPINGLABELS', 'SITEBUILDER', 'SITEBUILDER_STORE', 'SITELOCATIONALIASES', 'SOFTDESCRIPTORS', 'STACKABLEPROMOTIONS', 'STANDARDCOSTING', 'STATACCOUNTING', 'STOREPICKUP', 'SUBSCRIPTIONBILLING', 'SUITEANALYTICSCONNECT', 'SUITEAPPCONTROLCENTER', 'SUITECOMMERCE', 'SUITECOMMERCE_ADVANCED', 'SUITECOMMERCE_IN_STORE', 'SUITECOMMERCE_MY_ACCOUNT', 'SUITESIGNON', 'SUITE_OAX_CONNECTOR', 'SUPPLYALLOCATION', 'SUPPLYCHAINCONTROLTOWER', 'SUPPLYCHAINPREDICTEDRISKS', 'SUPPORT', 'TABLEAU', 'TBA', 'TEAMSELLING', 'TELEPHONY', 'TERMINATIONREASONTRACKING', 'TIMEBASEDPRICING', 'TIMEOFFMANAGEMENT', 'TIMETRACKING', 'TRANDELETIONREASONCODE', 'UNITSOFMEASURE', 'UPSELL', 'URLCOMPONENTALIASES', 'USR', 'VENDORACCESS', 'VENDORPREPAYMENTS', 'VENDORRETURNAUTHS', 'WBS', 'WEBAPPLICATIONS', 'WEBDUPLICATEEMAILMANAGEMENT', 'WEBHOSTING', 'WEBSERVICESEXTERNAL', 'WEBSITE', 'WEEKLYTIMESHEETS', 'WEEKLYTIMESHEETSNEWUI', 'WMSSYSTEM', 'WORKFLOW', 'WORKORDERS'],
```
Comment (omrilit) [inline packages/netsuite-adapter/src/autogen/types/enums.ts:304]: This modification will reflect in a fetch change for the user.
I think we should avoid doing it for now.
Anyway we'll have to regenerate types soon for the new API version (21.2) so it might be a good idea to wait with it IMO.
You can just revert this change manually. WDYT?

**Review by omrilit — APPROVED** [review]
Great improvement!
I added a single important comment so please address it before merging.
Just to make sure, did I miss any other implication on the customer on their next fetch?

**Comment by alonstern** [comment]
> Great improvement!
> I added a single important comment so please address it before merging.
> Just to make sure, did I miss any other implication on the customer on their next fetch?

It seems the the enums change was the only change
